.devcontainer/install-dependencies.sh

```diff
@@ -210,6 +210,15 @@ if should-install "$TOOL_DEST/cmctl"; then
     curl -L "https://github.com/jetstack/cert-manager/releases/latest/download/cmctl-${os}-${arch}.tar.gz" | tar -xz -C "$TOOL_DEST"
 fi
 
+BUILDX_DEST=$HOME/.docker/cli-plugins
+write-verbose "Checking for $BUILDX_DEST/docker-buildx"
+if should-install "$BUILDX_DEST/docker-buildx"; then
```

Comment:
We have the exact same issue with other installations, which is why this script has the `--skip-installed` flag so that installation can be forced.

(It's that way around so someone running the script with no parameters gets everything installed; `dev.sh` uses the flag to make setup quick if things are present.)